src/lib.rs

```diff
@@ -71,8 +71,10 @@ use std::fmt;
 use std::num::TryFromIntError;
 use std::time::Duration;
 
-pub mod api;
+#[cfg(feature = "async")]
+use r#async::Sleeper;
```

Comment:
@praveenperera I was running some standalone tests with custom runtimes: async-std and gloo-timers and noticed this. It should be `pub use` instead, right?

Reply:
@oleonardolima yup, made the change